File: PrismaApi/PrismaApi.Domain/Dtos/OptionDtos.cs
using System;

namespace PrismaApi.Domain.Dtos;

public class OptionDto
{
    public Guid Id { get; set; } = Guid.NewGuid();
    public string Name { get; set; } = string.Empty;
    public Guid DecisionId { get; set; }
    public double Utility { get; set; }
}

public class OptionIncomingDto : OptionDto
{
}

public class OptionOutgoingDto : OptionDto
{
}

File: PrismaFastApi/src/dtos/option_dtos.py
import uuid
from typing import Annotated
from pydantic import BaseModel, Field
from src.constants import DatabaseConstants


class OptionDto(BaseModel):
    id: uuid.UUID = Field(default_factory=uuid.uuid4)
    name: Annotated[str, Field(max_length=DatabaseConstants.MAX_SHORT_STRING_LENGTH.value)] = ""
    decision_id: uuid.UUID
    utility: float = 0.0


class OptionIncomingDto(OptionDto):
    pass


class OptionOutgoingDto(OptionDto):
    pass
